scripting/get5_apistats.sp

```diff
@@ -353,6 +353,17 @@ public void Get5_OnSeriesResult(MatchTeam seriesWinner, int team1MapScore, int t
   g_APIKeyCvar.SetString("");
 }
 
+public void Get5_OnSeriesResultWithoutWinners() {
```

Comment:
This forward doesn't exist. Why is this change even needed? ``Get5_OnSeriesResult`` already sends a request, it will just set the http "winner" param to "none".

Reply:
I've removed this. I understand now that `Get5_OnSeriesResult` do the work.I've removed this. I understand now that `Get5_OnSeriesResult` do the work.